### ISSUE TITLE
Localization not Working When apply current theme for Login

### ISSUE THREAD

**Issue by stantoxt** [OrchardCMS/OrchardCore#4949]

**Comment by hishamco** [comment]
Please list the steps to reproduce the issue, AFAIK there's an issue in data annotations, but this ma another localization issue

**Comment by stantoxt** [comment]
1. Add OrchardCore.Users/Account/Login to a theme 
2. Apply Login Using site theme for login page in Admin/Settings/LoginSettings
This will cause the localization problem.
![LoginLocalization](https://github.com/stantoxt/MarkdownResource/raw/master/pictures/addlogintotheme.png)

**Comment by hishamco** [comment]
Why you added the OrchardCore.Users/Account/Login to the theme?

**Comment by stantoxt** [comment]
I have to cutomize the login page.

**Comment by hishamco** [comment]
I see, let me check if I have a time ...

**Comment by stantoxt** [comment]
thanks @hishamco

**Comment by jtkech** [comment]
I think because in the related .po file the translations are tied to the users module. Maybe we could be more global for this kind of translations and / or when there are used in razor files that may be overridden.

    #. <h4>@T["Use a local account to log in."]</h4>
    #: OrchardCore.Users\Views\Account\Login.cshtml:21
    msgctxt "OrchardCore.Users.Views.Account.Login" <== this line
    msgid "Use a local account to log in."
    msgstr "Utilisez un compte local pour vous connecter."

Hmm, right now you would need to add some specific .po files in your theme.

**Comment by hishamco** [comment]
@jtkech the views will be overriden but I don't think the localization, furthermore the localization context still read the original resources

Correct me if I'm wrong?

**Comment by jtkech** [comment]
Not sure but if you explicitly tie it to a view `"OrchardCore.Users.Views.Account.Login"`

**Comment by hishamco** [comment]
Exactly, I need to check many localization scenarios after installing VS 16.4

**Comment by hishamco** [comment]
@stantoxt how OrchardCore.Users views work with you? I got **'Microsoft.NET.Sdk.Razor' SDK is required on Themes and Modules for Razor files to be precompiled.** error during build

**Comment by stantoxt** [comment]
@hishamco do you change <Project Sdk="Microsoft.NET.Sdk"> to <Project Sdk="Microsoft.NET.Sdk.Razor"> ?

**Comment by hishamco** [comment]
Nope, just copy the login view and `_ViewImports.cshtml`
